macosx/QuickLookExtension/PreviewViewController.h

```diff
@@ -0,0 +1,9 @@
+// This file Copyright © 2010-2022 Transmission authors and contributors.
+// It may be used under the MIT (SPDX: MIT) license.
+// License text can be found in the licenses/ folder.
+
+@import AppKit;
```

Comment:
This is probably the reason why `@import QuickLookUI` doesn't compile. Objective-C++ disallows `@import` statements in header files (save for a few select cases) and the errors when doing so are cryptic. I'd change this import to a preprocessor statement to see if it compiles then.